jsonrpc/Airplay.json

```diff
@@ -0,0 +1,101 @@
+{
+  "$schema": "interface.schema.json",
+  "jsonrpc": "2.0",
+  "info": {
+    "title": "Airplay API",
+    "class": "AirplayJSONRPC",
+    "description": "Airplay JSON-RPC interface"
+  },
+  "common": {
+    "$ref": "common.json"
+  },
+  "include": {
+    "statecontrol": {
+      "$ref": "StateControl.json#"
+    }
+  },
+  "properties": {
+    "visibility": {
+      "summary": "Current Airplay visibility",
+      "writeonly":true,
+      "events": [
+        "visibilitychange"
+      ],
+      "params": {
+        "type": "string",
+        "enum": [
+          "visible",
+          "hidden"
+        ]
+      }
+    }
+  },
+  "methods": {
+    "startAirplayApp": {
+      "summary": "Does Airplay system command",
+      "params": {
+        "type": "object",
+        "properties": {
+          "command": {
+            "description": "Command to send to Airplay",
+            "type": "string",
+            "example": "config"
+          }
+        }
+      },
+      "result": {
+        "type": "number"
+      },
+      "errors": [
+        {
+          "description": "Airplay is unavailable",
+          "$ref": "#/common/errors/unavailable"
+        },
+        {
+          "description": "Command is not set",
+          "$ref": "#/common/errors/unknownkey"
+        }
+      ]
+    },
+    "factoryreset": {
```

Comment:
Where does this factory reset happen? Does it reset the STB?

Reply:
There is Option under "Device Settings -> Factory reset". I believe user can choose to cleanup whole device (erase all sign-in info and app data) and restore device with original settings.